packages/eslint-plugin/src/rules/jsx-pragma/README.md

```diff
@@ -16,7 +16,7 @@ import { jsx } from '@compiled/react';
 ```
 
 For all Compiled usages, one of these should be used. See the
-[Emotion documentation](https://emotion.sh/docs/css-prop#jsx-pragma) for more details
+[React documentation](https://babeljs.io/docs/babel-plugin-transform-react-jsx) for more details
```

Comment:
I guess this should say Babel docs? Shame the React documentation doesn't specify it!

Reply:
argh that's what i get for multi-tasking too hard 😆

Reply:
but yeah the closest React docs I could find were https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html